proto/media.go

```diff
@@ -0,0 +1,33 @@
+package proto
+
+type RawImage []byte
+
+type MediaSet struct {
+	Object       MediaObject
+	Transcodings map[string]Transcoding
+}
+
+type MediaObject struct {
+	ID       string        `json:"id"`
+	Room     string        `json:"room"`
+	Storage  string        `json:"-"`
+	Uploader *IdentityView `json:"uploader"`
+	Created  Time          `json:"created"`
+	Uploaded Time          `json:"uploaded,omitempty"`
+}
+
+type Transcoding struct {
+	MediaID     string `json:"media_id"`
+	Name        string `json:"name"`
+	URI         string `json:"-"`
```

Comment:
The client will need to know where the image is, won't it? Is there another way to communicate that information?